main.go

```diff
@@ -85,6 +87,7 @@ func main() {
 	zapOptions.BindFlags(flag.CommandLine)
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&zapOptions)))
+	setupLog.Info("Build info", "version", BuildVersion, "date", BuildDate)
```

Comment:
I'd also be useful to log the MCAD and InstaScale versions, sourced from the Makefile as well.

Reply:
done